lib/msfl_visitors/collectors/chewy/term_filter.rb

```diff
@@ -0,0 +1,12 @@
+module MSFLVisitors
+  module Collectors
+    module Chewy
+      class TermFilter < String
```

Comment:
This can have some confusing ramifications, see here: http://words.steveklabnik.com/beware-subclassing-ruby-core-classes